src/mixins/document-mixin.ts

```diff
@@ -20,7 +20,8 @@ export default class DocumentMixin extends Vue {
 
   pdfjsLib: any
 
-  async created () {
+  // use beforeCreate() instead of created() to avoid type conflict with components that use this mixin
+  async beforeCreate (): Promise<void> {
```

Comment:
Interesting. Do you have a link to something discussing this?

Also, which type was causing the conflict?

Reply:
I actually don't know Sev. I brought this as part of me syncing the code from the legal name branch to here. It was a part of the initial legal name work that you've done in the legal name branch.

Unless I'm missing something, I didn't see any conflicts if we keep `created`. I just made this change to keep whatever you had there here.

Reply:
Keep whichever code is newest, and then run the unit tests and test this to make sure it's working as expected. Then resolve this comment. Thanks.

Reply:
Tested, works fine! Resolving.